app/views/layouts/head/_head.html.erb

```diff
@@ -1,7 +1,6 @@
 <head>
   <title>
-    <% if @title.present? %><%= @title %> | <% end %>
-    <%= t "general.project_shortname" %>
+    <%= t "general.project_name" %>
```

Comment:
this will mean that page titles will never reflect the contents, is that what you intended?

aka it used to be "About | African Poetics" but now all pages will just say "African Poetics"

Reply:
I made an issue for this https://github.com/CDRH/african_poetics/issues/27 but ran out of time. I'm hoping to look into it after Thursday